scripts/remove_after_use/fix_bad_embargo_termination_approvals.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import sys
+
+from website.app import setup_django
+setup_django()
+
+from django.db import transaction
+from osf.models import EmbargoTerminationApproval
+from osf.models import Sanction
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+from scripts import utils as scripts_utils
+
+
+def main(dry=True):
+    """
+    Fetches embargo termation approvals that were not marked as approved during the cron job.
+
+    The approve_embargo_termination script ran (and 48 hours had passed and there were users that hadn't approved)
+    The registrations attached to EmbargoTerminationApprovals were marked as public, and their embargo.state as `completed`,
+    but the corresponding embargo_termination_approval was left as `unapproved`.
+    """
+    bad_state_embargo_termination_approvals = EmbargoTerminationApproval.objects.filter(
+        embargoed_registration__is_public=True,
+        embargoed_registration__embargo__state=Sanction.COMPLETED,
+        state=Sanction.UNAPPROVED
```

Comment:
I don't have a current count, but this number shouldn't be very high

Reply:
177